.travis.yml

```diff
@@ -20,9 +20,10 @@ script:
   - "ansible-playbook -i tests/inventory tests/$PLAYBOOK --connection=local --sudo"
 
   # Run the role/playbook again, checking to make sure it's idempotent.
+  # Changed is 1 as the packages database will update
   - >
     ansible-playbook -i tests/inventory tests/$PLAYBOOK --connection=local --sudo
-    | grep -q 'changed=0.*failed=0'
+    | grep -q 'changed=1.*failed=0'
```

Comment:
@debo oh, agreed, I need to do investigation and now I have some free time, I can look into this the task that was causing the change is this:

```
- name: Ensure packages database is up to date
  become: yes
  apt: update_cache=yes
```